README.md

```diff
@@ -77,9 +77,9 @@ but is the result of exploring and testing new ideas.
 [Sven Nilsen](https://github.com/bvssvni/) started this project in early 2016.
 The idea was to make a simple, but convenient scripting language that integrated well with Rust.
 
-- During the first week of coding, it was discovered a way to do lifetime checking on function arguments
-- Dynamic modules with imported prelude were added to explore a different approach to organizing code
-- For nice error handling, added option, result and `?` operator
+- During the first week of coding, a way to do lifetime checking on function arguments was discovered
+- A different approach to code organization was explored by adding dynamic modules with imported preludes
```

Comment:
> by adding dynamic modules with imported preludes

I don't actually know what this is supposed to say so I guessed.

```
with imported preludes
with an imported prelude
via imported preludes
from imported preludes
by importing preludes
```

I don't think the wording is that idea either but I'm not actually sure what its trying to say so it's tricky.